src/react-components/home-root.js

```diff
@@ -184,6 +184,12 @@ class HomeRoot extends Component {
                   <a href="https://github.com/mozilla/hubs" rel="noreferrer noopener">
                     <FormattedMessage id="home.source_link" />
                   </a>
+                  <a href="/spoke" rel="noreferrer noopener">
+                    <FormattedMessage id="home.spoke_link" />
+                  </a>
+                  <a href="https://discord.gg/XzrGUY8" rel="noreferrer noopener">
```

Comment:
We use this discord.gg link in a ton of places now. Might be nice to make a https://hubs.mozilla.com/community redirect in reticulum at somepoint